code/modules/reagents/chemistry/reagents.dm

```diff
@@ -225,6 +226,20 @@ Primarily used in reagents/reaction_agents
 /datum/reagent/proc/get_taste_description(mob/living/taster)
 	return list("[taste_description]" = 1)
 
+/**
+ * Used when you want the default reagents purity to be equal to the normal effects
+ * (i.e. if default purity is 0.75, and your reacted purity is 1, then it will return 1.33)
+ *
+ * Arguments
+ * * normalise_num_to - what number/purity value you're normalising to. If blank it will default to the compile value of purity for this chem
+ * * _creation_purity - creation_purity override, if desired. This is the purity of the reagent that you're normalising from.
```

Comment:
Arguments should not be prefixed with underscores unless they are unused. Rename this to `creation_purity` and replace your existing use with just `src.creation_purity`.

Reply:
Huh, I missed that convention. Fixed.